target/linux/lantiq/image/tp-link.mk

```diff
@@ -63,3 +63,15 @@ define Device/tplink_vr200v
   SUPPORTED_DEVICES += VR200v
 endef
 TARGET_DEVICES += tplink_vr200v
+
+define Device/tplink_archer-d2
+  $(Device/lantiqTpLink)
+  DEVICE_MODEL := Archer D2
+  DEVICE_VARIANT := v1
+  SOC := ar10
```

Comment:
If we start to have separate SOC values for xrx200, the SOC variable should be set individually for all devices far clarity.